### ISSUE TITLE
Feature Request: make themes configurable (font, colors)

### ISSUE THREAD

**Issue by Dlanis** [openscopeproject/TrguiNG#86]
It would be nice if you could customize font and colors.

**Comment by qu1ck** [comment]
I don't plan adding this but you can easily configure styling as you wish if you compile the app yourself. Change fonts and colors here:
https://github.com/openscopeproject/TrguiNG/blob/master/src/components/mantinetheme.tsx

**Comment by Dlanis** [comment]
Know about it, and do it (1.), but I would prefer not to recompile for every theme change.

By the way, where in this file I can make font brighter for dark theme?

1. For some reason my system default font doesn't used, so I remove this garbage from fontFamily: `system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', ` to it work properly.

p.s. not English native

**Comment by qu1ck** [comment]
Add this to GlobalStyles 
```
            body: {
                color: theme.colorScheme === "dark" ? theme.white : theme.black,
            },
```
You can use other colors, read more [here](https://v6.mantine.dev/theming/colors/) and [here](https://v6.mantine.dev/theming/theme-object/).

**Comment by Dlanis** [comment]
Thanks!

**Comment by Dlanis** [comment]
Oh, I miss clicked.

**Comment by qu1ck** [comment]
I'll keep this as a wishlist issue but there won't be full theming controls, only ability to change font and probably main text color/background color.

**Comment by Dlanis** [comment]
Thanks! I think that's enough.

**Comment by qu1ck** [comment]
5a494c67c514e3c6138cf4a4c4dda90736fcc887 added configurable colors, font will come later.

**Comment by qu1ck** [comment]
Implemented in [v0.10.0](https://github.com/openscopeproject/TrguiNG/releases/tag/v0.10.0).

**Comment by Dlanis** [comment]
Thanks!
